### ISSUE TITLE
[Hunter][Spell] Steady Shot

### ISSUE THREAD

**Issue by Zabbil** [dalaranwow/dalaran-wow#4674]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

When player is not facing towards target, casting steady shot will gradually turn the caster towards the direction of target.  (Seems to be same mechanic as when spell channeling. however it’s not as smooth)

**Expected behaviour**: Tell us what should happen instead

The direction player is facing should never change when casting steady shot. 

**Steps to reproduce the problem**:

1. Cast steady shot when not facing a target head on, observe that your character is turning until you are facing towards target.
2. 
3. 

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Steady_Shot

« Patch 2.4.0 (25-Mar-2008): Hunters will no longer spin around if they cast Aimed Shot or Steady Shot while facing away from their target.»

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wotlk.openwow.com/spell=49052

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
> Current behaviour: Tell us what happens in detail (Where, When, What, Why, Who)
> 
> When player is not facing towards target, successful cast of Kill Shot will turn the caster towards the direction of target, (player will now face target head on).
> 
> Expected behaviour: Tell us what should happen instead
> 
> The direction player is facing should never change when casting Kill Shot.
> 
> Steps to reproduce the problem:
> 
> Cast Kill Shot successfully on a hostile target (when you are not facing the target head on), observe that after an successful Kill Shot cast, player is now facing target head on.
> 
> Include proofs for this behaviour
> 
> Possible same mechanic/script interaction as Steady Shot in issue #4674
> 
> I was not able to find any proof of this behavior. However I suspect that this has never been an issue, and therefore there are not any reports regarding this issue.
> 
> Regardless, instant spells should never interact with movement/direction of players.
> 
> Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills
> http://wotlk.openwow.com/spell=61006

Reported by @Zabbil

**Comment by Rushor** [comment]
those spells have SPELL_ATTR0_CAST_TRACK_TARGET which may needs to be removed here
